portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,35 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+    <body>
+    <header>
+        <nav class="menu">
+            <ul>
+                <li><a href ="#home">Home</a></li>
```

Comment:
HTML tag attributes like `href` here are normally written without any spaces between the attribute name, equals sign, and attribute values. So, this should be `href="#home"` rather than `href ="#home"`.